packages/venia-ui/lib/components/CheckoutPage/ShippingInformation/AddressForm/customerForm.gql.js

```diff
@@ -23,10 +23,8 @@ export const UPDATE_CUSTOMER_ADDRESS_MUTATION = gql`
         updateCustomerAddress(id: $addressId, input: $address)
             @connection(key: "updateCustomerAddress") {
             id
-            ...CustomerAddressFragment
```

Comment:
Shouldn't we be updating the UI when we make a change to the address?

Reply:
Added comment explaining the removal in 64cc0d5.